### PR TITLE
Newcomers_Guide.rst: fix a link typo

### LINKED ISSUE #3481: Newcomers_Guide.rst: There is a link typo
There is a link typo in coala/docs/Developers/Newcomers_Guide.rst:

### PR DESCRIPTION
**Pull request by gaocegege**
Fixes #3481 

Signed-off-by: Ce Gao <ce.gao@outlook.com>

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by gaocegege** [comment]
PTAL

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ As a preparation of joining the community you should find out what this project
 is about - if you didn't do this already. We highly recommend you
 `install coala <https://coala.io/install>`_ and use it on at least one of your
 projects. Also, we recommend that you read
-`Development Setup Notes <http://coala.io/devsetup>`
+`Development Setup Notes <http://coala.io/devsetup>`_
```
Comment (sils) [inline docs/Developers/Newcomers_Guide.rst]: I don't think we want that capitalised, other links aren't right?

**Comment by sils** [comment]
Please use imperative tense for the commit message, `Fix` instead of `Fixes`.

Other than those points this is good to go! Let us know if you need help of any kind.

If you're just looking for a brief contribution we can also fix it up for you, otherwise we'd like you to learn about our standards.

Thanks for your contribution and cheers!

**Comment by gaocegege** [comment]
Oh, Sorry for the bad commit messages. 

I'm looking for a code fixing tool just like coala and find this typo. I don't read the guide carefully. 😟 

I'll finish this PR after reading them.

**Comment by sils** [comment]
@gaocegege no issue, we're here to help!

**Comment by gaocegege** [comment]
PTAL :)

**Comment by sils** [comment]
ack https://github.com/coala/coala/pull/3477/commits/07ab1eca5f8223658a38db55544d9649ba04b6a0

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
Hey, this is your first contribution, right? Congrats and welcome to the coalaians! :) You've done your first step to become a coala developer.

Rultor will now automatically release your changes as a prerelease so users can experience the goodness of your changes right now.

We would really appreciate it if you could help us with

- [telling us about your experience](http://coala.io/newform)
- [some code reviews](http://coala.io/reviewing) for other newcomers
- fixing at least [a difficulty/low issue](http://coala.io/low)
- fill our [usability survey](http://coala.io/usability)

If you've done those and tell us about it so we can promote you to a coala developer!

We're all volunteers and we're currently struggling to keep this up. Helping all those newcomers is a lot of work and we do need your help so we can continue this for others!

**Comment by gaocegege** [comment]
Thank you, sils.

I would look through the code base and see what could I do next :)

**Pull request merged**